tests/unit/cli/test_cli_main.py

```diff
@@ -1,17 +1,43 @@
 """Tests for cli.main."""
 
 import fromconfig
+from fromconfig.cli.main import parse_args
+import sys
+from unittest.mock import patch
 
 import subprocess
 
 
 def capture(command):
-    """Utility to execute and capture the result of a commmand."""
+    """Utility to execute and capture the result of a command."""
     proc = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     out, err = proc.communicate()
     return out, err, proc.returncode
 
 
+def test_cli_parse_args():
+    """Test cli.parse_args."""
+    argv = ["fromconfig", "config.yaml", "--output", "/tmp", "-", "run"]
+    with patch.object(sys, "argv", argv):
+        paths, overrides, command = parse_args()
+        expected_paths = ["config.yaml"]
+        expected_overrides = {"output": "/tmp"}
+        expected_command = "run"
+        assert paths == expected_paths
+        assert overrides == expected_overrides
+        assert command == expected_command
+
+    argv = ["fromconfig", "config.yaml", "--output=/tmp", "-", "run"]
+    with patch.object(sys, "argv", argv):
+        paths, overrides, command = parse_args()
+        expected_paths = ["config.yaml"]
+        expected_overrides = {"output": "/tmp"}
+        expected_command = "run"
```

Comment:
Maybe it could be worth adding a unit test that covers the case of first example in the doc (https://fromconfig.github.io/#/). In such a case there are two "-".
I don't see why your code would not work in such setting but it is probably worth being sure that doc won't break.

Reply:
Good idea! The test case is added by the commit below.